src/main/resources/data/forbidden_arcanus/recipes/compat/botanypots/crop/golden_orchid.json

```diff
@@ -0,0 +1,34 @@
+{
+  "conditions": [
+    {
+      "type": "forge:mod_loaded",
+      "modid": "botanypots"
+    }
+  ],
+  "type":"botanypots:crop",
+  "seed":
+  {
+    "item":"forbidden_arcanus:golden_orchid_seeds"
+  },
+  "categories":
+  [
+    "magical_farmland"
+  ],
+  "growthTicks":1200,
+  "display":
+  {
+    "type":"botanypots:aging",
+    "block":"forbidden_arcanus:golden_orchid"
+  },
+  "drops":
+  [
+    {
+      "chance":1.00,
```

Comment:
With formatting issues, I mean for example missing spaces before the numbers like here.